doc/guides/porting/shields.rst

```diff
@@ -26,6 +27,11 @@ These files provides shield configuration as follows:
   format that is merged with the board's device tree information before
   compilation.
 
+* **<shield>.conf**: This file defines values for Kconfig symbols that are
+  required for default shield configuration. Please note that, to ease use with
```

Comment:
While it's polite, we don't need "please" in our documentation.  Delete "Please note that, " and start the sentence with "To ease use ..."